src/assets/styles/base.scss

```diff
@@ -150,10 +143,6 @@ p {
   white-space: pre-line;
 }
 
-.align-right {
```

Comment:
This was not used. And actually, there's a Vuetify helper class exactly for this.